src/operation/watch/mod.rs

```diff
@@ -0,0 +1,73 @@
+#[cfg(test)]
+mod test;
+use super::aggregate::{Aggregate, AggregateTarget};
```

Comment:
can we add line breaks between the various blocks in this file? (e.g. between the `mod` declaration and the `use`, between the `use` and the `struct`, etc.)

Reply:
fixed